src/main/java/mineverse/Aust1n46/chat/MineverseChat.java

```diff
@@ -1042,6 +1059,15 @@ public void run() {
 					p.getPlayer().sendMessage(LocalizedMessage.BLOCKING_MESSAGE.toString()
 							.replace("{player}", receiver));
 				}
+				if(identifier.equals("CustomError")) {
+					String message = msgin.readUTF();
+					String receiver = msgin.readUTF();
+					UUID sender = UUID.fromString(msgin.readUTF());
+					MineverseChatPlayer p = MineverseChatAPI.getOnlineMineverseChatPlayer(sender);
+					if(message.length()>0) {
+						p.getPlayer().sendMessage(message.replace("{player}", receiver));
+					}
+				}

```

Comment:
The concept of "custom error" seems too niche and unnecessary.  I would not have this.

Removing this greatly simplifies the event and this PR.